### PR TITLE
Room data is stored and reloaded

### PR DESCRIPTION
**Pull request by sjohnson31**

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -14,6 +14,8 @@ jobs:
           command: |
             python -m venv venv
             . ./venv/bin/activate
+            PYTHONPATH=./src/:$PYTHONPATH
```
Comment (sburba) [inline .circleci/config.yml]: Get rid of this ya dingus

File: api/src/room_store.py [hunk 2]
```diff
@@ -0,0 +1,23 @@
+import os
+import json
+
+
+class RoomStore:
+    def __init__(self):
+        self.path = os.environ['ROOM_STORE_DIR']
```
Comment (sburba) [inline api/src/room_store.py]: This should be a constructor argument. The only thing that should read environment variables is your entrypoint

File: api/src/room_store.py [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+        if not os.path.exists(self.path):
+            os.mkdir(self.path)
+
+    def get_all_room_ids(self) -> list:
+        return os.listdir(self.path)
+
+    def write_room_data(self, room: str, data: dict):
+        with open(f'{self.path}/{room}', 'w') as f:
```
Comment (sburba) [inline api/src/room_store.py]: This is scary, if room ids ever become user-controllable (via a bug or otherwise) then room ids like `../../../` will allow malicious actors to overwrite important files. You should verify that the resulting path is a child of the room directory before writing.

See here for an example: https://security.openstack.org/guidelines/dg_using-file-paths.html

File: api/src/room_store.py [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+class RoomStore:
+    def __init__(self):
+        self.path = os.environ['ROOM_STORE_DIR']
+        if not os.path.exists(self.path):
+            os.mkdir(self.path)
+
+    def get_all_room_ids(self) -> list:
+        return os.listdir(self.path)
+
+    def write_room_data(self, room: str, data: dict):
```
Comment (sburba) [inline api/src/room_store.py]: Everywhere else this is called `room_id`, probably makes sense to continue that pattern here

File: api/tests/test_game_state_server.py [hunk 5]
```diff
@@ -1,4 +1,4 @@
-from src.game_state_server import GameStateServer, Token
+import game_state_server as gss
```
Comment (sburba) [inline api/tests/test_game_state_server.py]: Why this change?

File: api/src/room_store.py (hunk 3 above)
Comment (sjohnson31) [reply api/src/room_store.py]: Good catch, thanks

File: api/src/room_store.py (hunk 4 above)
Comment (sjohnson31) [reply api/src/room_store.py]: I guess

File: api/tests/test_game_state_server.py (hunk 5 above)
Comment (sjohnson31) [reply api/tests/test_game_state_server.py]: Because it's better

File: api/src/room_store.py (hunk 2 above)
Comment (sjohnson31) [reply api/src/room_store.py]: Okay, fine
